PopPUNK/visualise.py

```diff
@@ -97,6 +97,10 @@ def get_options():
     iGroup.add_argument('--use-partial-query-graph',
                         help='File listing sequences in partial query graph after assignment',
                         default=None)
+    iGroup.add_argument('--extend-query-graph',
+                        help='Extend the partial query graph to include all co-clustered isolates',
```

Comment:
I don't understand this, which probably suggest more info is needed!

What are the co-clustered isolates?

When would/should this be used?

Reply:
Good point, fixed in 4e94a03 (and added missing argument).